browser-switch/src/main/java/com/braintreepayments/api/BrowserSwitchOptions.java

```diff
@@ -96,4 +98,13 @@ public Uri getUrl() {
     public String getReturnUrlScheme() {
         return returnUrlScheme;
     }
+
+    public boolean isLaunchAsNewTask() {
+        return launchAsNewTask;
+    }
+
+    public BrowserSwitchOptions launchAsNewTask(boolean launchAsNewTask) {
+        this.launchAsNewTask = launchAsNewTask;
+        return this;
+    }
```

Comment:
Should these just be a `get` and `set`?

Reply:
We may keep this for now. We have a pseudo-builder pattern in place for this value object that removes the `set` prefix from all the mutating functions, and the `is` prefix is a common accessor naming convention when the underlying type is a boolean.